hazelcast-hibernate/hazelcast-hibernate3/src/main/java/com/hazelcast/hibernate/instance/HazelcastAccessor.java

```diff
@@ -69,10 +69,6 @@ public static HazelcastInstance getHazelcastInstance(final SessionFactory sessio
     public static HazelcastInstance getHazelcastInstance(final SessionFactoryImplementor sessionFactory) {
         final Settings settings = sessionFactory.getSettings();
         final RegionFactory rf = settings.getRegionFactory();
-        if (rf == null) {
-            LOGGER.severe("Hibernate 2nd level cache has not been enabled!");
-            return null;
-        }
```

Comment:
I think we should not remove or move `HazelcastAccessor` class, some users might be using this utility class.